log/log.go

```diff
@@ -55,6 +58,7 @@ func CleanExit(format string, v ...interface{}) {
 func Err(format string, v ...interface{}) {
 	fmt.Fprint(Stderr, pretty.Colorize("{{.Red}}[ERROR]{{.Default}} "))
 	fmt.Fprintf(Stderr, appendNewLine(format), v...)
+	ErrorState = true
```

Comment:
Before I make changes, the state is not in error until one happens somewhere. Once that happens it change to an error state. Doesn't matter if it happens on one goroutine or another. The error state is global to the application.

What is gained from changing this to be a channel?